src/openbiolink/node.py

```diff
@@ -3,10 +3,11 @@
 
 
 class Node:
-    def __init__(self, id: str, type: NodeType, namespace: Namespace, name=""):
+    def __init__(self, id: str, type: NodeType, namespace: Namespace, name=None):
```

Comment:
Not sure what the impact of this change is, but it seems bad edge case to use an empty string if there's no data. Why don't all nodes have required names, though? This is something that would be really helpful for looking at the network and its edges

Reply:
I don't think the variable name is used ever, as far as I can tell. You mean a clear name like https://hpo.jax.org/app/browse/term/HP:0000118 "Phenotypic abnormality"?

Reply:
yeah, like having the label to go with each node is really powerful. The label for each node should be available in the source ontologies and terminologies

Reply:
I have to look into that, but this would indeed be a nice feature.